src/components/Calculator.js

```diff
@@ -1,62 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-    this.handleOperations = this.handleOperations.bind(this);
-  }
+function Calculator() {
+  const [object, setObject] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
 
-  handleOperations =(e) => {
-    this.setState((state) => calculate(state, e.target.textContent));
+  function handleOperations(e) {
+    const targetKey = (e.target.innerHTML);
+    setObject((object) => calculate(object, targetKey));
   }
 
-  render() {
-    const { total, next, operation } = this.state;
-
-    return (
-      <div className="calculus">
-        <div className="display">
-          {next || total || operation || 0 }
+  return (
+    <div className="calculus">
+      <div className="display">
+        { object }
+      </div>
+      <div className="All">
```

Comment:
- [ ] This error occurs when the app is run hence breaking it. Kindly fix it to have your project approved,
![image](https://user-images.githubusercontent.com/29071984/199873098-fa8a4b27-e086-4289-bb36-72e7d38d242c.png)